risk-assessment/src/service/githubService.ts

```diff
@@ -1,5 +1,50 @@
+import * as github from "@actions/github";
```

Comment:
I'm not sure how to test all of this but as always it would great to see test files.